tox.ini

```diff
@@ -19,8 +19,19 @@ commands =
 
 [testenv]
 deps =
-    -rrequirements_test.txt
+    pytest>=3.0
     dj18: Django>=1.8,<1.9
     dj19: Django>=1.9,<1.10
     dj110: Django>=1.10,<1.11
+    postgres: psycopg2>=2.6
```

Comment:
@mvantellingen In the case of DB adaptors you're probably right. Unfortunately with other apps it happened more than once that CI failed, not because of new bugs, but because of an implicit dependency change. Therefore we should following the zen of Python: Better explicit than implicit.